azerothcore/azerothcore-wotlk
# fix(core/Spell): Allow mages to refresh Living Bomb

Currently, mages can't refresh Living Bomb if the target already have the dot.

### How to reproduce the issue :
1. Use Living Bomb on a target
2. Try to re-use Living Bomb on the same target
3. Observe the error message

### Expected Blizzlike behaviour
Living Bomb should be reapplyable 

### Why this PR : 
This PR allow to refresh the duration of Living Bomb on the same target

I've didn't found a bad mage's video that refreshes Living Bomb on his target, but found some Fire Mage 3.3.5 guides that mentions : 
"Never refresh your LB (Living Bomb) before the explosion (except if I clearly tell you to)."
https://sites.google.com/site/pvemage335/fire/rotation-gameplay

"Living Bomb – every 12 seconds **after the explosion**"
https://www.gnarlyguides.com/wotlk/guides/pve-fire-mage/#plink-rotation

### How to test changes :
1. Use Living Bomb on a target
2. Re-use Living Bomb on the same target

### Target branch(es):

## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
